tests/parsers/test_sqla.py

```diff
@@ -35,13 +35,14 @@ def test_parse_filter():
     assert isinstance(f, BinaryExpression)
     assert d == ww
 
+
 def test_parse_filter_with_alias():
     ''' test a sqlalchemy filter parse with an aliased class '''
-    d = 'modela2.x > 5'
+    d = 'modela.x > 5 AND modela2.x < 3'
```

Comment:
I'd prefer to keep both tests.  Can you parametrize this test to test for both `modela2.x > 5` and `modela.x > 5 AND modela2.x < 3`?